README.md

```diff
@@ -20,11 +20,36 @@ You can add this Git repository as a remote OPAM repository:
 
   ```
   opam repo add xs-opam https://github.com/xapi-project/xs-opam.git
-  opam pin add lwt 2.7.1
   ```
 
-Due to constraints missing from old libraries, pinning the `lwt` package
-is necessary for certain configurations.
+## Building packages
+
+To install a package it's enough to run
+
+  ```
+  opam install $PKGNAME
```

Comment:
I know we talk about this later, but I'd say we also have to run `depext`, I tend to think about these commands together:
`opam depext package && opam install package`
I think for `xs-opam`, if we run install without depext, it's quite likely to fail. So maybe we could move that step up here from the "Neat tricks" section?